heft-plugins/heft-node-loc-plugin/src/utilities/Async.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ScopedLogger } from '@rushstack/heft';
+
+export class Async {
```

Comment:
Move to a file called `runWatcherWithErrorHandling` and `export function runWatcherWithErrorHandling`